_inc/client/state/survey/reducer.js

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
The state files are all in `_inc/client/state/survey`. Would it make more sense to have them in `_inc/client/state/disconnect-survey`? Especially since so much of this file is coupled to `disconnect`.